picard/config_upgrade.py

```diff
@@ -191,4 +198,5 @@ def upgrade_config():
     cfg.register_upgrade_hook(upgrade_to_v1_4_0_dev_2)
     cfg.register_upgrade_hook(upgrade_to_v1_4_0_dev_3)
     cfg.register_upgrade_hook(upgrade_to_v1_4_0_dev_4)
+    cfg.register_upgrade_hook(upgrade_to_v1_4_0_dev_5)
```

Comment:
This is a leftover from my first attempt, but i kept it since it logs a line about this change when running hooks.
I prefer to keep it.